experimental/dds/tree2/src/feature-libraries/forestSummarizerCodec.ts

```diff
@@ -0,0 +1,45 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { assert } from "@fluidframework/core-utils";
+import { ICodecOptions, IJsonCodec } from "../codec";
+import { fail } from "../util";
+import { FieldKey } from "../core";
+import { Format, Versioned, version } from "./forestSummarizerFormat";
+import { EncodedChunk } from "./chunked-forest";
+
+// TODO: As of now this codec does not contain that much logic on encoding, and mainly provides a codec which version stamps,
+// and validates the format of the data. Eventually we will want to add more logic in here.
+export function makeForestSummarizerCodec({
+	jsonValidator: validator,
+}: ICodecOptions): IJsonCodec<[FieldKey, EncodedChunk][], Format> {
+	const versionedValidator = validator.compile(Versioned);
+	const formatValidator = validator.compile(Format);
+	return {
```

Comment:
sorry missed this on first review, but you should add some tests for this file.

Reply:
Added unit tests